pages/apim/3.x/installation-guide/installation-guide-plugins.adoc

```diff
@@ -0,0 +1,187 @@
+= Installing and updating Plugins
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_installation_guide_plugins.html
+:page-folder: apim/overview
+:page-layout: apim3x
+:page-liquid:
+:page-description: Gravitee.io API Management - Installation Guide - Plugins
+:page-keywords: Gravitee.io, API Management, apim, guide, installation, upgrade, plugins
+:page-toc: true
+
+// author: Tom Geudens
+== Overview
+Plugins bring Gravitee APIM to life. They provide functionality ranging from connectors (eg. Kafka) to tracers (eg. Jaeger) and everything in between. A special mention goes to policies, plugins that provide you with options to shape and control the flow of your APIs.
+
+A lot of plugins are part of the Gravitee APIM installation. Ones that aren't generally applicable can be added. You can even write your own! This document explains how you can install and upgrade additional plugins.
+
+== In a nutshell - Option I
+. Get your plugin zip-file ready.
++
+The supported plugins for Gravitee APIM can be found link:https://download.gravitee.io/#graviteeio-apim/plugins/[here, window=\"_blank\"].
+
+. Drop the plugin zip-file in the plugins-folder of the relevant component. This could be the APIM REST API, the APIM Gateway or both!
+
+. Restart the component.
+
+NOTE: At this time installing and updating plugins requires a restart of the component.
+
+== In a nutshell - Option II
+. Get your plugin zip-file ready.
++
+The supported plugins for Gravitee APIM can be found link:https://download.gravitee.io/#graviteeio-apim/plugins/[here, window=\"_blank\"].
+
+. Drop the plugin zip-file in a plugins-folder of your own choosing.
+
+. Modify the `gravitee.yml` configuration file of the relevant component (APIM REST API / APIM Gateway / both) to take the additional folder into account
+
+. Restart the component.
+
+NOTE: At this time installing and updating plugins requires a restart of the component.
+
+== Details for .ZIP installations
+=== Option I
+. Locate the plugins folder of the component
```

Comment:
Why is there no `Get your plugin zip-file ready` on this section, like on the `In a nutshell` section ?